src/main/java/CarRacing/CarRacing.java

```diff
@@ -0,0 +1,55 @@
+package CarRacing;
```

Comment:
패키지 이름은 관례에 따라 전부 소문자로 작성해주시면 좋을 것 같습니다.
```suggestion
package carracing;
```

Reply:
수정 완료했습니다.